ResearchKit/Common/ORKHeadlineLabel.h

```diff
@@ -43,6 +44,8 @@ ORK_CLASS_AVAILABLE
 
 @property (nonatomic, assign) BOOL useSurveyMode;
 
+@property (nonatomic, strong, nullable) CEVRKTheme *cev_theme;
```

Comment:
No need to expose this as a public property in the header file; should be able to declare this in the .m file